README.md

```diff
@@ -59,6 +59,10 @@ The full path on the remote backup server where backups will be stored (all back
     backup_remote_host_name: ''
     backup_remote_host_key: ''
 
+Options to configure SSH connection options such as port (`-p port`). See [SSH manpages](http://man.openbsd.org/ssh) for more options.
```

Comment:
This comment needs to go _below_ the new variable, and the new variable needs to be after the next paragraph (in my docs, it goes var, then docs, then var, then docs).